docs/source/torch/features/feature_combination_matrix.md

```diff
@@ -0,0 +1,18 @@
+| Attention Data Parallelism | Yes               | Yes        | ---                        |                       |                 |     |                           |                           |               |                  |                |                        |                       |           |
+| Disaggregated Serving      | Yes               | Yes        | Yes                        | ---                   |                 |     |                           |                           |               |                  |                |                        |                       |           |
+| Chunked Prefill            | No                | Yes        | Yes                        | Untested              | ---             |     |                           |                           |               |                  |                |                        |                       |           |
+| MTP                        | Yes               | Yes        | Yes                        | Yes                   | Untested        | --- |                           |                           |               |                  |                |                        |                       |           |
+| EAGLE-3(One Model Engine)  | Yes               | Yes        | Yes                        | No                    | Untested        | No  | ---                       |                           |               |                  |                |                        |                       |           |
+| EAGLE-3(Two Model Engine)  | NO                | Yes        | Yes                        | No                    | Untested        | No  | No                        | ---                       |               |                  |                |                        |                       |           |
+| Torch Sampler              | Yes               | Yes        | Yes                        | Yes                   | Yes             | Yes | Yes                       | Yes                       | ---           |                  |                |                        |                       |           |
+| TLLM C++ Sampler           | Yes               | Yes        | Yes                        | No                    | Yes             | No  | No                        | No                        | No            | ---              |                |                        |                       |           |
+| KV Cache Reuse             |                   |            |                            |                       |                 |     |                           |                           |               |                  | ---            |                        |                       |           |
```

Comment:
Hi @byshiue , based on you testing, do you know if KV cache reuse can work with other features like overlap scheduler/CUDA graph/attention DP/disaggregated serving/etc? Thanks

Reply:
I have verified `kv cache reuse` with `overlap_scheduler` and `attention DP`. 

For `kv cache reuse` + `CUDA graph`, it could work, but the there are few performance issues. In fact, when `kv cache reuse` is enabled and reuse some caches, this means that there are few `context request` in the batch and hence we never use `CUDA graph` in such case.

For `disaggregated serving`, I don't verify.

For `Eagle3`, we are asked to disable the `kv cache reuse` now and I haven't tried to enable the `kv cache reuse` when we use `Eagle3`.

Reply:
+ @lfr-0531  and @mikeiovine  to comment, do you know if kv cache reuse can work with Eagle-3 or not?

Reply:
I would imagine that `kv cache reuse` is orthogonal to the choice of sampler, I am not aware of any code in the sampler where kv cache reuse is affected.